database/store.py

```diff
@@ -26,45 +27,196 @@ def applications_index(self) -> dict:
                 "id": application.get("id"),
                 "account_id": application.get("account_id"),
                 "status": application.get("status"),
+                "account_id": application.get("account_id"),
                 "fund_id": application.get("fund_id"),
                 "round_id": application.get("round_id"),
+                "project_name": application.get("project_name", ""),
+                "last_edited": application.get("last_edited"),
+                "started_at": application.get("started_at"),
```

Comment:
⛏️ Can this not just be initialised here?

Reply:
started_at could be yes, last_edited currently presuming null to start with and then has a datetime when some answers have been added (rather than the form minted), and project_name is null to start with and then that get's added (currently) on the first page of the form.